Package.swift

```diff
@@ -1,18 +1,22 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.3
+
 import PackageDescription
 
 let package = Package(
     name: "AcknowList",
+    defaultLocalization: "en",
     platforms: [
-        .iOS(.v8), .tvOS(.v9)
+        .iOS(.v9), .tvOS(.v9), .macOS(.v10_15)
```

Comment:
What’s the reason for adding macOS? I’d love to support it too, but as of today, the library requires `UIKit`.